lib/msal-core/src/ServerRequestParameters.ts

```diff
@@ -54,7 +54,11 @@ export class ServerRequestParameters {
   constructor (authority: Authority, clientId: string, scope: Array<string>, responseType: string, redirectUri: string, state: string) {
     this.authorityInstance = authority;
     this.clientId = clientId;
-    this.scopes = scope;
+    if (!scope) {
+      this.scopes = [clientId];
+    } else {
+      this.scopes = scope.slice();
```

Comment:
what does slice with no params do here? return back the original array?

Reply:
Yes that's correct:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice

Reply:
If it does nothing why call it

Reply:
So that any mutations on it don't mutate the passed in array. 

You can do this via ES6 like so:

```js
this.scopes = [ ...scope ];
```

Reply:
It returns a copy instead of the original, so we don't have any mutation. @jasonnutter is it better to use ES6 notation than .slice()?

Reply:
It looks cooler :) should be functionally equivalent.